Makefile

```diff
@@ -258,3 +258,14 @@ gather-sysinfo-tests: build-gather-sysinfo
 .PHONY: render-sync
 render-sync: build
 	hack/render-sync.sh
+
+
+.PHONY: pao-build-e2e
+pao-build-e2e:
+	@for suite in 0_config 1_performance 2_performance_update 3_performance_status 4_latency 5_latency_testing 6_mustgather_testing 7_performance_kubelet_node 8_performance_workloadhints 9_reboot; do \
```

Comment:
I would look in the `functests` directory in general because when others adding more suites they might miss updating this target